lib/Runtime/Language/InterpreterHandlerAsmJs.inl

```diff
@@ -180,6 +180,10 @@ EXDEF2_WMS( F1toF1Mem        , Nearest_Flt      , Wasm::WasmMath::Nearest<float>
 EXDEF2_WMS( D1toD1Mem        , Trunc_Db         , Wasm::WasmMath::Trunc<double>                      )
 EXDEF2_WMS( D1toD1Mem        , Nearest_Db       , Wasm::WasmMath::Nearest<double>                    )
 EXDEF2_WMS( I1toI1Mem        , PopCnt_Int       , ::Math::PopCnt32                                   )
+EXDEF2_WMS( D1toI1Scr        , Conv_Check_DTI   , (OP_TruncWithCheck<int,INT_MIN,INT_MAX>)           )
```

Comment:
yes since it contains a ",". Initially, I used `#define COMMA ,`  but it doesn't work with clang.